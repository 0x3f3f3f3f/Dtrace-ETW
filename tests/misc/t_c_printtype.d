/* printtype.exe */
pid$target::ff_getgameid:entry
/next == 0/
{
	print(*args[0]);

	printf("\n");
	next = 1;
}

pid$target::ff_getpartysize:entry
/next == 1/
{
	print(*args[0]);
	printf("\n");
	next = 2;
}

pid$target::ff_getsummons:entry
/next == 2/
{
	print(*args[0]);
	printf("\n");
	exit(0);
}