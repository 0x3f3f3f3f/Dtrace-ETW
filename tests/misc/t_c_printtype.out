 0  65823               ff_getgameid:entry final_fantasy_info_t {
    int ff_gameid = 0x1
    int ff_partysize = 0x4
    int ff_hassummons = 0
}

  0  65824            ff_getpartysize:entry final_fantasy_info_t {
    int ff_gameid = 0xa
    int ff_partysize = 0x3
    int ff_hassummons = 0x1
}

  0  65825              ff_getsummons:entry final_fantasy_info_t {
    int ff_gameid = 0x6
    int ff_partysize = 0x4
    int ff_hassummons = 0x1
